restcomm/restcomm.interpreter/src/test/java/org/restcomm/connect/interpreter/GatherSpeechTest.java

```diff
@@ -527,6 +527,25 @@ public void testIgnoreRcmlFromPartialCallback() throws Exception {
         };
     }
 
+    @Test
+    @SuppressWarnings("unchecked")
+    public void testDtmfUsingSipInfo() throws Exception {
```

Comment:
name is test case is not corresponding with issue. issue was in `dtmf by mediaserver` not `dtmf using sip-info`